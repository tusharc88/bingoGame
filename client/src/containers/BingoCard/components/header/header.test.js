import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Header from './index';

test('Renders with default props', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
