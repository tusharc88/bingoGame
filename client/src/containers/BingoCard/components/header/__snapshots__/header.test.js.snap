// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders with default props 1`] = `
.c1 {
  width: 100%;
}

.c4 {
  height: 50px;
  width: 75px;
  background-color: #edf0f3;
  text-align: center;
  vertical-align: middle;
  font-size: 25px;
  line-height: 50px;
  border-radius: 5px;
  border: 2px solid #3ecedf;
}

.c8 {
  height: 30px;
  width: 45px;
  background-color: #edf0f3;
  text-align: center;
  vertical-align: middle;
  font-size: 15px;
  line-height: 30px;
  border-radius: 5px;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.c6 {
  font-size: 15 px;
  width: 95%;
  margin: 10px;
}

.c7 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin-left: 25px;
}

.c3 {
  font-size: 25px;
  line-height: 50px;
  margin-right: 25px;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  width: 95%;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <div
      className="c2"
    >
      <div
        className="c3"
      >
        Last Ball
      </div>
      <div
        className="c4"
      >
        1
      </div>
    </div>
  </div>
  <div
    className="c1"
  >
    <div
      className="c5"
    >
      <div
        className="c6"
      >
        PREVIOUS BALLS
      </div>
      <div
        className="c7"
      >
        <div
          className="c8"
        >
          1
        </div>
        <div
          className="c8"
        >
          2
        </div>
        <div
          className="c8"
        >
          3
        </div>
        <div
          className="c8"
        >
          4
        </div>
      </div>
    </div>
  </div>
</div>
`;
